qa/L0_infer/test.sh

```diff
@@ -157,7 +158,7 @@ for TARGET in cpu gpu; do
               sed -i "s/dims: \[ 1 \]/dims: \[ -1, -1 \]/" config.pbtxt)
     fi
 
-    run_server
+    run_server_v2
```

Comment:
Change the utilities so there is only the v2 versions... and rename them to not have _v2 suffix.

Reply:
Done